hunting/json.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+from dataclasses import asdict
+import json
+from pathlib import Path
+import click
+from .definitions import Hunt
+from .utils import load_index_file, load_toml
+
+class JSONGenerator:
```

Comment:
there is really no need to create for a class here: the grouping of the logic is achieved by using a separate module (though, it's better to rename to avoid the confusion with default `json` package) and we have no use for state beyond this module.

I suggest refactoring this as separate stateless functions